content/TrueCommand/TCReleaseNotes.md

```diff
@@ -23,8 +23,30 @@ aliases:
 | 2.3.0 | Tag | TBD |
 | 2.3.0 | Release | TBD |
 
+## 2.2.1 Hotpatch
```

Comment:
We have been saying "hotpatch" internally, but I am not sure thats the wording we want to use

Its just another dot release

Reply:
fixed!